### ISSUE TITLE
jpg mode

### ISSUE THREAD

**Issue by RedSquirrel87** [RedSquirrel87/manga-downloader#268]
**[Original report](https://bitbucket.org/Red_Squirrel/manga-downloader/issues/268) by Anonymous.**

----------------------------------------

When a image is already in jpg, what does the jpg mode do? I guess this mode is useful for decreasing the file size, but if an image is already in jpg, then it's basically useless, you'll only lose quality by using it again. So, here's my suggestion:

* When a image is already in jpg, leave it like that, don't convert it again. Make it so that jpg mode only modifies non-jpg images (if it's not like this yet);
* If an image is in png and its file size is like 200 kb, it's nonsense losing quality by converting it again. Enable a check so that images smaller than a user-chosen file size don't get converted. 

In other words, I'm talking about a change of purpose of the jpg mode, making it more like a "smaller file size mode". A random user doesn't know what jpg even is and ends up downloading the chapters in lower quality for no reason at all. If, instead, this became a "smaller file size mode", then it would make more sense. Just my humbly opinion, in any case.
